New geo module

rangerdawavs opened the pull request:
changed previous geo module, as well as added variables for speed calculation that seem to have gotten stuck somewhere in this mess of commits
pending review
camera functionality tested and completed, need to check if the correct info is still being sent to the arduino.

rangerdawavs committed:
added new geo module

rangerdawavs committed:
added second group of changes for new geo module

rangerdawavs commented:
most changes were renaming commands in 'main', the changes to the camera module are for type consistency.

leobrenes reviewed (changes requested):
why is there no y2=int(y2) in camera_m.py?
para sub_prot_rasp_main.py, we were calculating the angle betweent the y axis and the current vector, should we change the arduino code to use the x axis or change this one?
the part with ceta and dev with respect to time should change when merged with the main, which will already have the calculus part merged before.

leobrenes committed:
Added Calculus 

in what used to be lines 16-18 I deleted some variables @rangerthedawavs. Their purpose is fulfilled by calculusm_1
Still missing the pixels to cm mapping for position data. seems to be ready for merging to geogebra but this feature must be added before merging with master.

leobrenes committed:
Merge pull request #19 from rangerdawavs/leobrenes-Geogebra-w/-calc

Added Calculus

leobrenes committed:
Merge branch 'master' into new-geo-module
